### PR TITLE
A small update to the "fr" lang.

### PR DESCRIPTION
**Pull request by ernest33**
Hi, 
Just an update to the french translation.

### CODE REVIEW

File: src/locales/lang/fr.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
...
   "plugins.search.placeholder": "Taper pour rechercher",
   "plugins.unsplash.photoLink": "Photo",
-  "plugins.weather.high": "High",
-  "plugins.weather.low": "Low",
-  "settings": "Settings",
-  "settings.translationCredits": "Traduction française par Orfaon",
-  "widgets": "Widgets"
+  "plugins.weather.high": "Max",
+  "plugins.weather.low": "Min",
+  "settings": "Préférences",
```
Comment (CheriseCodes) [inline src/locales/lang/fr.json:16]: When I change my phone's settings to (European) French, "Settings" is translated to "Réglages". I'd guess this is the terrminology most Francophone people are used to.

File: src/locales/lang/fr.json [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
+  "settings.translationCredits": "Traduction française par Orfaon/ernest33",
+  "widgets": "Gadgets"
```
Comment (CheriseCodes) [inline src/locales/lang/fr.json:18]: My phone also uses the term "Widgets".

**Review by CheriseCodes — CHANGES_REQUESTED** [review]
Based on navigating an iPhone in European French, some terms may need to be updated to be more natural for French speakers.

**Comment by joelshepherd** [comment]
Given I am not a French speaker, I might wait and for a consensus on this.

File: src/locales/lang/fr.json (hunk 1 above)
Comment (ernest33) [reply src/locales/lang/fr.json:16]: > 
> 
> When I change my phone's settings to (European) French, "Settings" is translated to "Réglages". I'd guess this is the terrminology most Francophone people are used to.

Yes this is used too :-)

File: src/locales/lang/fr.json (hunk 2 above)
Comment (ernest33) [reply src/locales/lang/fr.json:18]: > 
> 
> My phone also uses the term "Widgets".

Yes that's true, but this is an "Anglicism" ;-)

**Pull request merged**